src/clj/rems/application/model.clj

```diff
@@ -566,13 +453,13 @@
             enrich-members)))
 
 (defn enrich-workflow-handlers [application get-workflow]
-  (if (= :workflow/dynamic (get-in application [:application/workflow :workflow/type]))
-    (let [workflow (get-workflow (get-in application [:application/workflow :workflow/id]))
-          handlers (get-in workflow [:workflow :handlers])]
+  (let [workflow (get-workflow (get-in application [:application/workflow :workflow/id]))
+        handlers (get-in workflow [:workflow :handlers])]
+    (if handlers
```

Comment:
is this `if` necessary? if there are no handlers, we'll just have an empty vec, right?